### PR TITLE
fix(curriculum): clean-up Project Euler 462-480

### PR DESCRIPTION
**Pull request by gikf**
- Cleans description.
- Changes generic `eulerXXX` function name.
- Adds images.
- Problem 461 was reworked, so I haven't touched it.
- Problem 462 had expected answer `Infinity`, this doesn't seem to be correct. I've replaced it with the result, which can be found as an answer for this problem.
- Problems 463 and 473 were using MathJax already, but I've cleaned that a bit.
- Uses MathJax, a lot.
- Related to #40896.

Needed before merging:
- [x] Merged PR adding needed images to CDN https://github.com/freeCodeCamp/cdn/pull/105.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-462-permutation-of-3-smooth-numbers.md [hunk 1]
```diff
@@ -8,41 +8,47 @@ dashedName: problem-462-permutation-of-3-smooth-numbers
...
 -   1, 2, 4, 3, 9, 8, 16, 6, 18, 12.
 
 This is not a valid permutation because 12 comes before its divisor 6.
 
 -   1, 2, 4, 3, 9, 8, 12, 16, 6, 18.
 
-We can verify that F(6) = 5, F(8) = 9, F(20) = 450 and F(1000) ≈ 8.8521816557e21. Find F(1018). Give as your answer its scientific notation rounded to ten digits after the decimal point. When giving your answer, use a lowercase e to separate mantissa and exponent. E.g. if the answer is 112,233,445,566,778,899 then the answer format would be 1.1223344557e17.
+We can verify that $F(6) = 5$, $F(8) = 9$, $F(20) = 450$ and $F(1000) ≈ 8.8521816557e\\,21$.
+
+Find $F({10}^{18})$. Give as your answer as a string in its scientific notation rounded to ten digits after the decimal point. When giving your answer, use a lowercase e to separate mantissa and exponent. E.g. if the answer is $112\\,233\\,445\\,566\\,778\\,899$ then the answer format would be `1.1223344557e17`.
```
Comment (moT01) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-462-permutation-of-3-smooth-numbers.md:25]:
```suggestion
Find $F({10}^{18})$. Give as your answer as a string in its scientific notation rounded to ten digits after the decimal point. When giving your answer, use a lowercase `e` to separate mantissa and exponent. E.g. if the answer is $112\\,233\\,445\\,566\\,778\\,899$ then the answer format would be `1.1223344557e17`.
```

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-464-mbius-function-and-intervals.md [hunk 2]
```diff
@@ -8,39 +8,46 @@ dashedName: problem-464-mbius-function-and-intervals
...
 
-The Möbius function, denoted μ(n), is defined as:
+The Möbius function, denoted $μ(n)$, is defined as:
 
-μ(n) = (-1)ω(n) if n is squarefree (where ω(n) is the number of distinct prime factors of n)
+- $μ(n) = (-1)^{ω(n)}$ if $n$ is squarefree (where $ω(n)$ is the number of distinct prime factors of $n$)
+- $μ(n) = 0$ if $n$ is not squarefree.
 
-μ(n) = 0 if n is not squarefree.
+Let $P(a, b)$ be the number of integers $n$ in the interval [a, b] such that $μ(n) = 1$.
```
Comment (moT01) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-464-mbius-function-and-intervals.md:16]:
```suggestion
Let $P(a, b)$ be the number of integers $n$ in the interval $[a, b]$ such that $μ(n) = 1$.
```

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-464-mbius-function-and-intervals.md [hunk 3]
```diff
@@ -8,39 +8,46 @@ dashedName: problem-464-mbius-function-and-intervals
...
 
-Let P(a,b) be the number of integers n in the interval \[a,b] such that μ(n) = 1. Let N(a,b) be the number of integers n in the interval \[a,b] such that μ(n) = -1. For example, P(2,10) = 2 and N(2,10) = 4.
+Let $N(a, b)$ be the number of integers $n$ in the interval [a, b] such that $μ(n) = -1$.
```
Comment (moT01) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-464-mbius-function-and-intervals.md:18]:
```suggestion
Let $N(a, b)$ be the number of integers $n$ in the interval $[a, b]$ such that $μ(n) = -1$.
```

**Review by moT01 — COMMENTED** [review]
Wow - thanks for all your hard work on these @gikf 🎉 I can't believe how fast you got through them all.

**Comment by gikf** [comment]
> Wow - thanks for all your hard work on these @gikf  I can't believe how fast you got through them all.

Thanks for all the reviews and merges @ieahleen @moT01 @nhcarrigan @raisedadead @scissorsneedfoodtoo @Sembauke :tada: :tada: :tada:

**Comment by Sembauke** [comment]
> Thanks for all the reviews and merges @ieahleen @moT01 @nhcarrigan @raisedadead @scissorsneedfoodtoo @Sembauke 🎉 🎉 🎉

No Problem! Happy to help.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by naomi-lgbt — APPROVED** [review]
Woohoo!

**Pull request merged**